src/leappto/cli.py

```diff
@@ -7,7 +8,12 @@
 from pwd import getpwuid
 from subprocess import Popen, PIPE
 from collections import OrderedDict
-from leappto.providers.libvirt_provider import LibvirtMachineProvider, LibvirtMachine
+from leappto import Machine
+from leappto.driver import LocalDriver
+from leappto.driver.ssh import LocalDriver, SSHDriver
```

Comment:
`LocalDriver` currently gets imported twice - I'm guessing the first one is the intended one.